src/search/landmarks/landmark_cost_partitioning_heuristic.cc

```diff
@@ -18,13 +18,14 @@ namespace landmarks {
 LandmarkCostPartitioningHeuristic::LandmarkCostPartitioningHeuristic(
     const plugins::Options &opts)
     : LandmarkHeuristic(opts),
-      cost_partitioning_strategy(opts.get<CostPartitioningStrategy>("cost_partitioning")) {
+      cost_partitioning_strategy(
+          opts.get<CostPartitioningStrategy>("cost_partitioning")) {
```

Comment:
Rename to `cost_partitioning_method` and `CostPartitioningMethod`? The `Algorithm` suffix would be even better, but I think we should use that name for the base class.

Reply:
Also, I think we should pass `cost_partitioning_method`, `alm` and `lpsolver` to the `set_cost_partitioning_algorithm` method and not store `cost_partitioning_method` as a member variable.

Reply:
I applied the changes in the first comment. Regarding the second one: I also removed the member. All options you mention are (still) passed to the function within the `opts` parameter for now. Would you also change the function header?

Reply:
I think it's bad practice to pass around the options variable, but let's not change this here.